src/Tests/OneExplorer/ConfigObject.test.ts

```diff
@@ -0,0 +1,235 @@
+          assert.isObject(configObj!.obj);
+          assert.isArray(configObj!.obj.baseModels);
+          assert.isArray(configObj!.obj.derivedModels);
+
+          assert.strictEqual(configObj!.obj.baseModels.length, 1);
+          assert.strictEqual(configObj!.obj.baseModels[0].fsPath, modelPath);
+          assert.strictEqual(configObj!.obj.derivedModels.length, 0);
+        }
+      });
+
+      test('Returns parsed object with derivedModels', function() {
+        const configName = 'model.cfg';
+        const baseModelName = 'model.tflite';
+        const derivedModelName1 = 'model.circle';
+        const derivedModelName2 = 'model.q8.circle';
+
+        const content = `
+[onecc]
+one-import-tflite=True
+one-quantize=True
+[one-import-tflite]
+input_path=${baseModelName}
+output_path=${derivedModelName1}
+[one-quantize]
+input_path=${derivedModelName1}
+output_path=${derivedModelName2}
+        `;
+
+        // Write a file inside a temp directory
+        testBuilder.writeFileSync(configName, content);
+
+        // Get file paths inside the temp directory
+        const configPath = testBuilder.getPath(configName);
+        const baseModelPath = testBuilder.getPath(baseModelName);
+        const derivedModelPath1 = testBuilder.getPath(derivedModelName1);
+        const derivedModelPath2 = testBuilder.getPath(derivedModelName2);
+
+        const configObj = ConfigObj.createConfigObj(vscode.Uri.file(configPath));
+
+        // Validation
+        {
+          assert.isNotNull(configObj);
+          assert.isNotNull(configObj!.rawObj);
+          assert.isNotNull(configObj!.obj);
+
+          assert.isObject(configObj!.rawObj);
+          assert.isObject(configObj!.obj);
+          assert.isArray(configObj!.obj.baseModels);
+          assert.isArray(configObj!.obj.derivedModels);
+
+          assert.strictEqual(configObj!.obj.baseModels.length, 1);
+          assert.strictEqual(configObj!.obj.baseModels[0].fsPath, baseModelPath);
+          assert.strictEqual(configObj!.obj.derivedModels.length, 2);
+
+          assert.isTrue(configObj!.obj.derivedModels.map(derivedModel => derivedModel.fsPath)
+                            .includes(derivedModelPath1));
+          assert.isTrue(configObj!.obj.derivedModels.map(derivedModel => derivedModel.fsPath)
+                            .includes(derivedModelPath2));
+        }
+      });
+
+      test('Parse config with detouring paths', function() {
+        const configName = 'model.cfg';
+        const baseModelName = 'model.tflite';
+        const derivedModelName1 = 'model.circle';
+        const derivedModelName2 = 'model.q8.circle';
+
+        // Detouring paths
+        const content = `
+[onecc]
+one-import-tflite=True
+one-quantize=True
+[one-import-tflite]
+input_path=/dummy/dummy/../../${baseModelName}
+output_path=/dummy/dummy/../../${derivedModelName1}
+[one-quantize]
+input_path=////${derivedModelName1}
+output_path=/dummy/dummy/../..//${derivedModelName2}
```

Comment:
These example seems confusing to me. 🐱 
Since path starts from `/`, I think these should be interpreted as absolute paths.

For example,
```
$ pwd
/home/eric/m/work/3/ONE-vscode
$ realpath -m ../dummy/dummy/..//model.tflite   # relative path
/home/eric/m/work/3/dummy/model.tflite
$ realpath -m ///dummy/dummy/..//model.tflite  # absolute path
/dummy/model.tflite
```

When I try to print `configObj!.obj.baseModels` and `configObj!.obj.derivedModels` at L226, they are all in `/tmp/one-vscode.test/ConfigObject/4` and I am not sure they are correct. (e.g., `'/tmp/one-vscode.test/ConfigObject/ConfigObject/4/model.tflite'`)

When a path is in absolute path, I think we should not join like L145.

Reply:
Sure. Actually that went through my mind which I just ignored... Let me work on that.

Reply:
> When I try to print configObj!.obj.baseModels and configObj!.obj.derivedModels at L226, they are all in /tmp/one-vscode.test/ConfigObject/4 and I am not sure they are correct. (e.g., '/tmp/one-vscode.test/ConfigObject/ConfigObject/4/model.tflite')

It's right. I created a temporal directory with numbers for the test debug.

Reply:
I have updated the code to work by [Taking as an absolute path when it starts with /](https://github.com/Samsung/ONE-vscode/pull/681/commits/2899ea77e812533a91e90e905e464fb630726a9f)